### PR TITLE
Fix flaky tests by increasing the timeout for a task to complete.

### PR DESCRIPTION
**Pull request by jodavis**
Also realized that the TaskAssert helper class wasn't actually testing
anything, so I fixed that. Then I needed to fix another test that wasn't
testing correctly.

### CODE REVIEW

**Comment by jodavis** [comment]
Addresses https://github.com/aspnet/Home/issues/3382

File: test/Microsoft.VisualStudio.Web.BrowserLink.Test/TaskAssert.cs [hunk 1]
```diff
@@ -113,22 +105,7 @@ private static void ThrowFailure(Exception exception, params string[] failureMes
         {
```
Comment (prafullbhosale) [inline test/Microsoft.VisualStudio.Web.BrowserLink.Test/TaskAssert.cs]: Can this method be removed? `exception` parameter is not being used. All callers can switch to using the other overload of this method.

**Comment by natemcmaster** [comment]
Good to merge?

**Comment by prafullbhosale** [comment]
Yes. Is `release/2.2` still the right branch for this?

**Comment by natemcmaster** [comment]
Yes.

**Pull request merged**